### ISSUE TITLE
no Studio notification for permission request response

### ISSUE THREAD

**Issue by helenvarley** [upstage-org/upstage#1310]
to test the permissions system, i requested permission from vicki to use her media item WhaleBone.jpg. she received the request, and denied it (i'm trying not to take that personally), & i received her response via email, but there is no notification in the Studio notifications interface.

i guess it's not essential to get a notification there as well as an email, but if that isn't working then what is that notifications interface for? it is potentially a place for the acknowledgement information, or to find your way to where that is (see issue #1258)

![Screenshot from 2024-07-29 21:20:04](https://github.com/user-attachments/assets/4a84c733-d834-4686-99be-a960fb2159d6)

**Comment by vickis** [comment]
Message ID: ***@***.***>ha ha all other requests have been granted (and I did say if you asked again I would grant it ;) it was to understand what happened when the media item is rejected eg. could you still select it? 

It is a useful aspect for ensuring appropriate acknowledgements and avoiding copyright infringement

**Comment by helenvarley** [comment]
>  it was to understand what happened when the media item is rejected eg. could you still select it?  
no, i still have the "request permission" button, so that's as it should be.

but this issue is specifically about the lack of anything appearing in the Notifications in the Studio - the megaphone icon next to your player name in the top bar.

**Comment by helenvarley** [comment]
this has been fixed
